test/Turn-test.js

```diff
@@ -0,0 +1,58 @@
+const chai = require('chai');
+const expect = chai.expect;
+const Card = require('../src/Card');
+const Turn = require('../src/Turn');
+
+describe('Turn', () => {
+  let card, turn, card1, turn1;
+
+  beforeEach(() => {
+    card = new Card(1, 'What allows you to define a set of related information using key-value pairs?', ['object', 'array', 'function'], 'object');
+    turn = new Turn('array', card);
+    card1 = new Card(2, 'What is a comma-separated list of related values?', ['array', 'object', 'function'], 'array')
+    turn1 = new Turn('array', card1);
+  });
+
+  it('should be a function', () => {
+    expect(Turn).to.be.a('function');
+  });
+
+  it('should instantiate a new turn', () => {
+    expect(turn).to.be.an.instanceof(Turn);
+  });
+
+  it('should have a user guess', () => {
+    expect(turn.guess).to.equal('array');
+    expect(turn.guess).to.be.a('string');
```

Comment:
i don't know if this is me being picky, but i'd move the data type expect  on line 26 to 25  and then check for the answer to 26.

Reply:
Good call, will do!